src/BrandedNavBar/MobileMenu.tsx

```diff
@@ -17,107 +18,63 @@ const BrandingWrap = styled.div(({ theme }) => ({
   "&:visited": {
-    color: theme.colors[color] || color,
+    color: theme.colors.darkBlue,
   },
-  "&:hover": {
-    color: "#434d59", // darkGrey
-    background: "#f0f2f5", // whiteGrey
+  width: "100%",
+  borderRadius: "0",
+  transition: ".2s",
+  "&:hover, &:focus": {
+    outline: "none",
+    color: theme.colors.blackBlue,
+    backgroundColor: theme.colors.whiteGrey,
+    cursor: "pointer",
   },
-});
-
-type ApplyMenuLinkStylesProps = {
-  layer?: number;
-  color?: string;
-  hoverColor?: string;
-  hoverBackground?: string;
-  theme?: DefaultNDSThemeType;
-};
-
-const ApplyMenuLinkStyles = styled.li(
-  ({
-    color = "white",
-    hoverColor = "lightBlue",
-    hoverBackground = "white",
-    layer = 0,
-    theme,
-  }: ApplyMenuLinkStylesProps) => ({
-    display: "block",
-    "button, a": {
-      ...getSharedStyles({ color, layer, theme }),
-      textDecoration: "none",
-      "&:hover, &:focus": {
-        outline: "none",
-        color: theme.colors[hoverColor] || hoverColor,
-        backgroundColor: theme.colors[hoverBackground] || hoverBackground,
-      },
-      "&:disabled": {
-        opacity: ".5",
-      },
-      "&:focus": {
-        boxShadow: theme.shadows.focus,
-      },
-    },
-  })
-);
+  "&:focus": {
+    boxShadow: theme.shadows.focus,
+  },
+  "&:disabled": {
+    opacity: ".5",
+  },
+}));
 
-type MenuLinkProps = {
-  layer?: number;
-  color?: string;
-  hoverColor?: string;
-  hoverBackground?: string;
-  theme?: DefaultNDSThemeType;
-};
+const TopLevelText = styled(Text)(({ theme }) => ({
+  ...getSharedStyles(theme),
+  color: theme.colors.blackBlue,
+}));
 
-const MenuLink = styled.a(
-  ({ color, hoverColor, hoverBackground, layer, theme }: MenuLinkProps) => ({
-    ...getSharedStyles({ color, layer, theme }),
-    width: "100%",
-    borderRadius: "0",
-    transition: ".2s",
-    "&:hover, &:focus": {
-      outline: "none",
-      color: themeGet(`colors.${hoverColor}`, hoverColor)(hoverColor),
-      backgroundColor: themeGet(
-        `colors.${hoverBackground}`,
-        hoverBackground
-      )(hoverBackground),
-      cursor: "pointer",
-    },
-    "&:focus": {
-      boxShadow: theme.shadows.focus,
-    },
-    "&:disabled": {
-      opacity: ".5",
-    },
-  })
-);
+const getSharedStyles = (theme) => ({
+  display: "block",
+  textDecoration: "none",
+  border: "none",
+  backgroundColor: "transparent",
+  fontSize: theme.fontSizes.large,
+  fontWeight: theme.fontWeights.medium,
+  lineHeight: theme.lineHeights.heading3,
+  padding: `${theme.space.x1} ${theme.space.x3}`,
+  paddingLeft: getPaddingLeft(0),
+});
 
-type MenuTextProps = {
-  textColor?: string;
+type ApplyIndentProps = {
   layer?: number;
   theme?: DefaultNDSThemeType;
+  key?: string;
 };
 
-const MenuText = styled.li(({ textColor, layer, theme }: MenuTextProps) => ({
-  ...getSharedStyles({ color: textColor, layer, theme }),
+const ApplyIndent = styled.li(({ layer, theme }: ApplyIndentProps) => ({
```

Comment:
@sskirby I don't like this name. It should make it obvious that it changes the child elements and it also does more than that!